contexts/DonationContext/src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -116,7 +116,7 @@ private function getPhysicalAddressFromRequest( AddDonationRequest $request ): D
 	private function getNameFromRequest( AddDonationRequest $request ): DonorName {
 		$name = $request->donorIsCompany() ? DonorName::newCompanyName() : DonorName::newPrivatePersonName();
 
-		$name->setSalutation( $request->getDonorSalutation() );
+		$name->setSalutation( $request->donorIsCompany() ? Donation::COMPANY_SALUTATION : $request->getDonorSalutation() );
```

Comment:
So even if a salutation is provided for a company, it will be changed? Is that intentional?